docs/devs/readme.md

```diff
@@ -1 +1,6 @@
-#
\ No newline at end of file
+# Miscellaneous points developers should know and follow
+
+- Ensure dev dependencies are installed in your Aeon Python environment
+- Ensure pre-commit hooks are installed in your Aeon Python environment: run `pre-commit install` in the activated environment
+- Use [pyan3](https://github.com/Technologicat/pyan) for generating callgraphs
```

Comment:
Can we elaborate here on what we are supposed to do with the output of `pyan3`? I can see how call graphs might be useful when reading the code base in some cases, but do we need to generate them when developing? Or is it more of a general tooling recommendation?

Reply:
This was more of a reminder to myself. I found pyan when I wanted to generate diagrams for callgraphs for readers.

It's just a random point, but I can move it, any thoughts on where it should live?

Reply:
Ok, decided just to remove this point from here